### ISSUE TITLE
Crates new versions for `stable2409-1` haven't been published yet

### ISSUE THREAD

**Issue by pandres95** [polkadot-fellows/runtimes#493]
Just that. Here's a sample error I'm having when trying to update to `stable2409-1` using `psvm`:

```
    Updating crates.io index
error: failed to select a version for the requirement `emulated-integration-tests-common = "^14.1.0"`
candidate versions found which didn't match: 14.0.0, 13.0.0, 11.0.0, ...
location searched: crates.io index
required by package `asset-hub-kusama-emulated-chain v1.0.0 (/Users/pandres95/Documents/Virto/Development/Fellowship/runtimes/integration-tests/emulated/chains/parachains/assets/asset-hub-kusama)`
```

It's the same error with every single crate that was updated in `stable2409-1`

**Comment by bkchr** [comment]
Sounds like a `psvm` error?

**Comment by pandres95** [comment]
Not sure about that @bkchr. Or maybe? I ran the update using this command:

```sh
psvm --version stable2409-1
```

Then, it bumped versions to some crates (need to check again which ones, but there are at least 6 or 7 of them). After that, I started getting those `cargo` errors, just to find out that those versions didn't exist at all on `crates.io`. 😅

**Comment by xlc** [comment]
The patch versions are not released because the release for polkadot-sdk is broken. we have to fix it.

Essentially it is possible for example in stable2407, a crate is published with version 1.2.3 and someone made a minor change on master to bump it to 1.2.4. Then when stable2409 is published, 1.2.4 will be published.
Now we want to backport a change back to stable2407 and release stable2407-1. What version is it going to be? it can't be 1.2.4 because that's already published. now what? and people decided to ignore the issue 💁‍♂️

**Comment by pandres95** [comment]
I just figured out I raised the issue in the wrong repository :')

Closed in favour of [polkadot-sdk#6206](https://github.com/paritytech/polkadot-sdk/issues/6206)
